package.json

```diff
@@ -84,8 +90,12 @@
     "serve": "gatsby serve",
     "test": "jest",
     "test:coverage": "jest --coverage",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open",
+    "test:e2e:run": "start-server-and-test serve http://localhost:8000 cy:run",
```

Comment:
```suggestion
    "pretest:e2e:run": "yarn build",
    "test:e2e:run": "start-server-and-test serve http://localhost:8000 cy:run",
```

Reply:
This change is needed to make sure that the app is built before it is served.
The CircleCI build failed because the application was served before it was built.